### PR TITLE
Added Svelte Package and NPM in README

### PR DESCRIPTION
**Pull request by haseeb-heaven**
Added Svelte Package and NPM package along with bun

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,12 +114,18 @@ To install Devika, follow these steps:
...
+   Install the Svelte package required
+   use command `npm install @sveltejs/vite-plugin-svelte` to install it
    ```
    cd ui/
    bun run dev
    ```
-7. Access the Devika web interface by opening a browser and navigating to `http://127.0.0.1:3000`.
+   or use npm
+   ```
+   nmp run dev
```
Comment (Zaid-maker) [inline README.md:126]: spelling mistake!

**Review by Zaid-maker — CHANGES_REQUESTED** [review]

**Comment by haseeb-heaven** [comment]
@Zaid-maker Fixed the spelling now check

**Review by Zaid-maker — APPROVED** [review]
All good!

File: README.md [hunk 2]
```diff
@@ -114,12 +114,18 @@ To install Devika, follow these steps:
    ```
    python devika.py
    ```
-6. Compile and run the UI server:
+6. Compile and run the UI server:</br>
+   Install the Svelte package required
+   use command `npm install @sveltejs/vite-plugin-svelte` to install it
```
Comment (123vivekr) [inline README.md:119]: `@sveltejs/vite-plugin-svelte` is already present in ui/package.json
`bun install` should install it. Am I missing something?

Comment (haseeb-heaven) [reply README.md:119]: It didn’t installed for me i used pip install -r requirements file

Comment (123vivekr) [reply README.md:124]: You'd have to do `bun install` or `npm install`
`@sveltejs/vite-plugin-svelte` is a package in npm not pip

Comment (haseeb-heaven) [reply README.md:124]: But on Readme it was not mentioned now maybe its updated

**Pull request closed without merging**